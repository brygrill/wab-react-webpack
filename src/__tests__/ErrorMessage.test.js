import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ErrorMessage from '../components/ErrorMessage';

describe('<ErrorMessage />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ErrorMessage />, div);
  });

  it('displays error message', () => {
    const tree = renderer.create(<ErrorMessage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
