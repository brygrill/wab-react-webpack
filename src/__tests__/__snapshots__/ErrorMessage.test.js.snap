// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> displays error message 1`] = `
<div>
  <p>
    Sorry â€” something has gone wrong.
  </p>
</div>
`;
